table-cat/table-cat.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+"""
+Script to concatenate multiple files (table-format) with common header,
+also works for gzip files, with output in non-gzip table format.
+"""
+import sys
+import argparse
+import gzip
+
+parser = argparse.ArgumentParser()
+parser.add_argument("tables", nargs='*',
```

Comment:
Change '*' to '+'. '+' indicates that at least 1 argument is required and prints a nice error message if none is provided. Instead, with '*' the program crashes.